client/dashboard/store-performance/index.js

```diff
@@ -2,9 +2,15 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 import { ToggleControl } from '@wordpress/components';
 import { Component, Fragment } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withDispatch } from '@wordpress/data';
+import { getCurrentDates, appendTimestamp, getDateParamsFromQuery } from '@woocommerce/date';
```

Comment:
Minor but it seems like in some components we have been adding a `/** WooCommerce Dependencies ` block for internal packages.